conf/polarion_tools.yaml

```diff
@@ -0,0 +1,111 @@
+default_fields:
+    arch: ""
+    assignee: ""
+    automation_script: ""
+    caseautomation: automated
+    casecomponent: "-"
+    caseimportance: high
+    caselevel: component
+    caseposneg: positive
+    customerscenario: ""
+    description: ""
+    endsin: ""
+    expectedResults: ""
+    initialEstimate: ""
+    legacytest: ""
+    linkedWorkItems: ""
+    multiproduct: ""
+    reqverify: ""
+    setup: ""
+    startsin: ""
+    subcomponent: ""
+    subtype1: "-"
+    subtype2: "-"
+    tags: ""
+    teardown: ""
+    testSteps: ""
+    testtier: ""
+    testtype: functional
+    title: ""
+    upstream: ""
+    work_item_id: ""
+
+custom_fields:
+    - arch
+    - automation_script
+    - caseautomation
+    - casecomponent
+    - caseimportance
+    - caselevel
+    - caseposneg
+    - customerscenario
+    - endsin
+    - legacytest
+    - multiproduct
+    - reqverify
+    - setup
+    - startsin
+    - subcomponent
+    - subtype1
+    - subtype2
+    - tags
+    - teardown
+    - testtier
+    - testtype
+    - upstream
+
+docstrings:
+    required_fields:
+        - assignee
+        - initialEstimate
```

Comment:
I believe we want caseimportance to be required as well.

Reply:
Ignore me, default is `high`, we'll leave that for now, can easily change in the future.